templates/tools/dockerfile/test/cxx_ubuntu1604_x64/Dockerfile.template

```diff
@@ -39,7 +39,7 @@
   # The clang-3.6 symlink for the default clang version was added
   # to Ubuntu 16.04 recently, so make sure it's installed.
   # Also install clang3.7.
-  RUN apt-get update && apt-get -y install clang-3.6 clang-3.7 && apt-get clean
+  RUN apt-get update && apt-get -y install gcc-4.8 g++-4.8 clang-3.6 clang-3.7 && apt-get clean
```

Comment:
I think we also need gcc-4.8-multilib and g++-4.8-multilib for this to work, or at least to be consistent with everything else